portfolio/src/main/webapp/script.js

```diff
@@ -106,23 +86,32 @@ function start() {
         }, 1000);
 
         currentQuestion += 1;
-        if (currentQuestion >= questions.length) {
-            let endText = document.createElement("div");
-            endText.innerText = `Your score: ` + score +
-                `/` + questions.length + `. Thank you for playing! By the way, that's true, the right answer is always right :)`;
-            endText.classList.add("quizEnd");
-            quizContainer.innerHTML = "";
-            quizContainer.append(endText);
-            return;
-        }
-        scoreElement.innerHTML = "Current score: " + score;
-        question.innerHTML = questions[currentQuestion];
-        leftButton.innerHTML = leftAnswers[currentQuestion];
-        rightButton.innerHTML = rightAnswers[currentQuestion];
+        displayQuestion(currentQuestion, questions, leftAnswers, rightAnswers, score, scoreElement, leftButton, rightButton, question, quizContainer);
     });
+    // display first question
+    displayQuestion(currentQuestion, questions, leftAnswers, rightAnswers, score, scoreElement, leftButton, rightButton, question, quizContainer);
+}
 
+function displayQuestion(currentQuestion, questions, leftAnswers, rightAnswers, score, scoreElement, leftButton, rightButton, question, quizContainer) {
+    // if it was the last question
+    if (currentQuestion == questions.length) {
```

Comment:
I heard it's better to use '===' for comparison. Please see https://stackoverflow.com/q/359494, for example.

Reply:
Yeah, it seems like it is always better to use '===' if applicable. Changed it!